### PR TITLE
longpolling support

### PR DESCRIPTION
**Pull request by yelizariev**
the update is needed when workers > 0

### CODE REVIEW

File: envoy.yaml [hunk 1]
```diff
@@ -11,6 +11,12 @@ static_resources:
     lb_policy: round_robin
     name: target_odoo
     type: strict_dns
+  - connect_timeout: 0.25s
+    hosts:
+    - socket_address: {address: odoo, port_value: 8072}
+    lb_policy: round_robin
+    name: target_odoo_longpolling
+    type: strict_dns
```
Comment (blaggacao) [inline envoy.yaml]: Just to understand (I'm leanring)... What does this do?

File: envoy.yaml [hunk 2]
```diff
@@ -119,6 +125,8 @@ static_resources:
             - domains: [odoo.*]
               name: local_service
               routes:
+              - match: {prefix: /longpolling/}
+                route: {cluster: target_odoo_longpolling, timeout: 90s}
```
Comment (blaggacao) [inline envoy.yaml]: Standard timeout would be too short, I assume? - Odoo uses somewhat above 60, is 90 to largely cover the odoo timeout?

File: envoy.yaml (hunk 1 above)
Comment (yelizariev) [reply envoy.yaml]: I thought you know it :smile:

File: envoy.yaml (hunk 2 above)
Comment (yelizariev) [reply envoy.yaml]: Odoo breaks longpolling each 50 seconds. Shall I set something like 55 seconds?

Comment (blaggacao) [reply envoy.yaml]: Yes, + do a comment for the rational?

**Comment by blaggacao** [comment]
Thanks a lot! :heart_eyes:

**Pull request merged**